### ISSUE TITLE
Video storage and features editing

### ISSUE THREAD

**Issue by narcisgarcia** [Chocobozzz/PeerTube#3362]
With Peertube 2.4.0 once a video is uploaded and re/encoded, neither administrator nor owner can't customize files resolutions or encodings to optimize storages and bandwidth.
My feature requests about this:

A single video storage should be editable (both by administrators and owner), for example, with removing one of its files/resolutions and leave others, and commanding it to be re-encoded to a lower resolution or different format (webm-vp9-ogg / MP4-mpeg4-aac / aac).

This could include a preview of each stored file for that video, being useful to compare results.

Split from #3294

**Comment by geopayme** [comment]
I would add an option to expand storage using Digital Ocean Volumes, I have added 100G volume on a instance but I can't select the storage its defaulted. I use YunoHost and I see the resources but there is no way to select where the videos can be stored! I say a must feature for those who use Virtual services. I have deployed 8 instances so far and the struggle is space management! 
![image](https://user-images.githubusercontent.com/8299759/103161428-635eab00-47ea-11eb-8a4f-15a67d91246d.png)
